mlflow/sklearn.py

```diff
@@ -57,8 +58,9 @@ def save_model(sk_model, path, conda_env=None, mlflow_model=Model(),
     :param conda_env: Either a dictionary representation of a Conda environment or the path to a
                       Conda environment yaml file. If provided, this decribes the environment
                       this model should be run in. At minimum, it should specify the dependencies
-                      contained in ``mlflow.sklearn.DEFAULT_CONDA_ENV``. If `None`, the default
-                      ``mlflow.sklearn.DEFAULT_CONDA_ENV`` environment will be added to the model.
+                      contained in ``mlflow.sklearn.get_default_conda_env()``. If `None`, the
```

Comment:
Nit: can we change this to 
```
:func:`get_default_conda_env()`
```
along with the other 3 instances?

Reply:
I've filed https://github.com/mlflow/mlflow/pull/1238/ which makes this change for all relevant flavors, including `sklearn`.